src/actions/components.js

```diff
@@ -99,3 +101,15 @@ export const handleClose = () => ({
   type: HANDLE_CLOSE,
   payload: false,
 });
+
+export const updatePosition = (id, x, y) => ({
```

Comment:
You might want to consider using object parameters e.g {id, x, y} so that in the future if someone mistakenly changes the order of input the application is not broken.